src/hmn_extras.c

```diff
@@ -131,6 +131,9 @@ void libxmp_hmn_extras_process_fx(struct context_data *ctx, struct channel_data
 {
 	switch (fxt) {
 	case FX_MEGAARP:
+		/* Not sure if this is correct... */
```

Comment:
is there a way we can verify this?

Reply:
I don't have the speical version of Noisetracker used for this format *or* an emulation setup that can run older Amiga trackers right now. Claudio might know, but considering that there are only 16 arpeggio patterns, I'm not sure there is a correct handling for the higher values.

Reply:
OK then, let's get this in, hopefully nothing breaks.